File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/PregledPostVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Ordinacija_API.ModelVM
{
    public class PregledPostVM
    {

        public string  Datum { get; set; }
        public string Vrijeme { get; set; }
        public string Opis { get; set; }
        public int PacijentID { get; set; }
        public int ZaposlenikID { get; set; }
        public int UslugaID { get; set; }


    }
}

File: app/src/main/java/com/example/ordinacija/Data/PregledPostVM.java
package com.example.ordinacija.Data;

import java.util.Date;

public class PregledPostVM {
    public String Datum;
    public String Vrijeme;
    public String Opis;
    public int ZaposlenikID;
    public int PacijentID;
    public int UslugaID;

    public PregledPostVM(String Datum,String Vrijeme,String Opis,int ZaposlenikID,int PacijentID,int UslugaID){
    this.Datum=Datum;
    this.Vrijeme=Vrijeme;
    this.Opis=Opis;
    this.ZaposlenikID=ZaposlenikID;
    this.PacijentID=PacijentID;
    this.UslugaID=UslugaID;
    }
}
